src/Windows/Avalonia.Win32/Avalonia.Win32.csproj

```diff
@@ -35,6 +37,8 @@
     <NoWarn>$(NoWarn);CA1416</NoWarn>
   </PropertyGroup>
   <ItemGroup>
+    <SupportedPlatform Remove="@(SupportedPlatform)" />
```

Comment:
Would be nice to have some comments with explanation

Reply:
Added a comment. Also see
https://github.com/dotnet/designs/blob/main/accepted/2020/platform-exclusion/platform-exclusion.md